src/main/java/com/internet/shop/dao/OrderDao.java

```diff
@@ -0,0 +1,21 @@
+package com.internet.shop.dao;
+
+import com.internet.shop.model.Order;
+import java.util.List;
+import java.util.Optional;
+
+public interface OrderDao {
+    Order create(Order order);
+
+    Optional<Order> get(Long id);
+
+    List<Order> getAll();
+
+    Order update(Order order);
+
+    boolean deleteById(Long id);
+
+    boolean delete(Order order);
+
+    public List<Order> getUserOrders(Long userId);
```

Comment:
```suggestion
   List<Order> getUserOrders(Long userId);
```
public is access modifier by default in interfaces